src/dev/mocha/run_mocha_cli.js

```diff
@@ -0,0 +1,75 @@
+import { resolve } from 'path';
+
+import getopts from 'getopts';
+import globby from 'globby';
+
+export function runMochaCli() {
+  const opts = getopts(process.argv.slice(2), {
+    alias: {
+      t: 'timeout',
+    },
+    boolean: [
+      'no-timeouts'
+    ],
+  });
+
+  const runInBand = process.execArgv.some(arg => {
+    switch (arg) {
+      case '--inspect':
+      case '--inspect-brk':
+        return true;
+    }
+  });
+
+  // ensure that mocha exits when test have completed
+  process.argv.push('--exit');
+
+  // ensure that mocha requires the babel-register script
+  process.argv.push('--require', require.resolve('../../babel-register'));
+
+  // set default test timeout
+  if (opts.timeout == null && !opts['no-timeouts']) {
+    if (runInBand) {
+      process.argv.push('--no-timeouts');
+    } else {
+      process.argv.push('--timeout', '10000');
+    }
+  }
+
+  // set default slow timeout
+  if (opts.slow == null) {
+    process.argv.push('--slow', '5000');
+  }
+
+  // set default reporter
+  if (opts.reporter == null) {
+    process.argv.push('--reporter', require.resolve('./server_junit_reporter'));
+  }
+
+  // set default test files
+  if (!opts._.length) {
+    globby.sync([
+      'src/**/__tests__/**/*.js',
+      'packages/kbn-datemath/test/**/*.js',
+      'packages/kbn-dev-utils/src/**/__tests__/**/*.js',
```

Comment:
__note (not related to this pr):__ btw, with `kbn-dev-utils/src/**/__tests__` you excluded _transpiled_ tests from `target` folder which is good, but wondering if `babelrc` from `kbn-dev-utils` should exclude `__tests__` folder as well.

Reply:
Perhaps, I was mostly trying to avoid searching in the `kbn-dev-utils/node_modules`.

Reply:
Haha, for some reason I thought that `node_modules` is always ignored (iirc TS ecxludes it by default unless you override `exclude` section)

Reply:
I don't think that'd make sense for a general purpose tool like [globby](https://github.com/sindresorhus/globby)

Reply:
Ugh, sorry, I was thinking about `babel` when I was talking about `node_modules` and `target` - just ignore me :man_facepalming: